src/Components/articles-page.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react"
+import { useEffect } from "react"
+import { getArticles } from "../../api"
+import ArticlesList from "./articles-list"
+
+function ArticlesPage(){
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(()=>{
+        getArticles()
+        .then((response)=>{
+            setArticles(response.articles)
+            
+        })
+    }, [])
```

Comment:
It looks like it would make sense to have this useEffect inside the articleList component because that's the only place it's being used